src/ui/components/form/form-header/FormHeader.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Typography } from '@material-ui/core';
+import { withStyles, StyleRules } from '@material-ui/core/styles';
+
+import { WithStyles } from '../../../../interfaces/form/form.interface';
+import themeColors from '../../../global/themeColors';
+
+const styles = (): StyleRules => ({
+  title: {
+    color: themeColors.black,
+    fontSize: '24px',
+    fontWeight: 800,
+  },
+});
+
+interface FormHeaderProps extends WithStyles<typeof styles> {
+  headerText: string;
+}
+
+const FormHeader = ({ classes, headerText }: FormHeaderProps): JSX.Element => {
```

Comment:
Is there anything specific to forms in this component? Should we rename it to be something more generic so it can be used outside of forms also?